Enterprise-CMCS/managed-care-review
# MCR-3826 ensuring Q&A sidenav renders where expected

## Summary
This PR sets up the sideNav to appear for state users where expected, mainly in the edit submission form.
#### Related issues
[MCR-3826](https://jiraent.cms.gov/browse/MCR-3826)

## QA guidance

1. Access an unlocked rate as a state user
2. The sideNav should be present across all pages of the form
3. The Q&A button should navigate you to the Q&A page - caqn check the id in the url to verify
4.  The submission sideNav button should direct you towards the review-and-submit page for an unlocked form
